portal-api/src/v1/validation/validateIds.js

```diff
@@ -44,10 +44,22 @@ const isValidCompaniesHouseNumber = (companiesHouseNumber) => isValidRegex(COMPA
  */
 const isValidBankId = (bankId) => isValidRegex(/^\d+$/, bankId);
 
+/**
+ * isValidMonth
+ * checks if month is an integer between 1 and 12
+ * @param {Number} month
+ * @returns {Boolean} asserts if regex is matched
```

Comment:
Would use lower case `number` and `boolean` here as primitives are used in these functions.